tests/e2e/vsphere_volume_expansion.go

```diff
@@ -1500,7 +1500,7 @@ var _ = ginkgo.Describe("Volume Expansion Test", func() {
 		gomega.Expect(err).NotTo(gomega.HaveOccurred())
 		gomega.Expect(pvclaim).NotTo(gomega.BeNil())
 
-		if featureEnabled {
+		if !featureEnabled {
```

Comment:
This feature gate is about cnsNewSyncFSS. Should the message on line 1504 mention this feature gate?